src/resources/items/armor.ts

```diff
@@ -70,5 +70,16 @@ export const armor = armorObject({
 		level: 2,
 		slotsUsed: 2,
 		itemLevel: 3
-	}
+	},
+	tactical_bodyarmor: {
+		type: 'Body Armor',
+		name: 'Tactical_Bodyarmor',
+		icon: '<:U_shield:601366669474136074>',
+		description: 'Body armor that looks like it can stop a couple of bullets',
+		aliases: ['Tactical'],
```

Comment:
aliases should be all lowercase letters
```suggestion
		aliases: ['tactical'],
```